src/docs/development/ui/animations/staggered-animations.md

```diff
@@ -316,7 +316,7 @@ class _StaggerDemoState extends State<StaggerDemo> with TickerProviderStateMixin
       [[highlight]]await _controller.forward().orCancel;[[/highlight]]
       [[highlight]]await _controller.reverse().orCancel;[[/highlight]]
     } on TickerCanceled {
-      // the animation got canceled, probably because we were disposed
+      // the animation got canceled, probably because it was disposed of
```

Comment:
It looked strange to me, but I may be wrong as I am not a native speaker of English.